lib/containers/container_images.pm

```diff
@@ -183,6 +184,32 @@ sub test_opensuse_based_image {
     }
 }
 
+sub verify_userid_on_container {
+    my ($runtime, $image) = @_;
+    record_info "host uid", script_output "echo \$UID";
+
+    record_info "root user inside", "process runs in the user's host uid";
+    my $cid = script_output "$runtime run -d --rm --name test1 $image sleep infinity";
```

Comment:
this is repeating blocks of code , maybe extract it to a function ?

Reply:
i had some thinking around it but i really dont think it is worth it. And i do not think it is something that it will be reused.